cellmlmanip/model.py

```diff
@@ -555,21 +558,29 @@ def get_symbol_by_ontology_term(self, namespace_uri, local_name):
             raise ValueError('Multiple variables annotated with {%s}%s' %
                              (namespace_uri, local_name))
 
-    def _get_symbols_by_rdf(self, predicate, object_=None):
+    def get_rdf_annotations(self, subject=None, predicate=None, object=None):
```

Comment:
Use `object_` rather than `object` to avoid shadowing the built-in.

Reply:
o yes